src/library_glfw.js

```diff
@@ -598,7 +598,7 @@ var LibraryGLFW = {
 #if USE_GLFW == 3
         {{{ makeDynCall('viii', 'GLFW.active.windowSizeFunc') }}}(GLFW.active.id, GLFW.active.width, GLFW.active.height);
 #endif
-      });
```

Comment:
The idea is that if the runtime is no longer active we do want to exit here.  See the `keepRuntimeAlive` function.. for some reason it must b returning false in your use case, and its probably worth looking into why.